packages/components/InputPanel/InputPanel.tsx

```diff
@@ -0,0 +1,51 @@
+import { Flex, Grid } from '@chakra-ui/react';
+
+import { pass } from '@packages/utils/pass';
+
+import BaseButton from '../BaseButton';
+
+import { NumericButton } from './NumericButton';
+
+const generateNumberList = (max = 10) => {
+  return Array(max)
+    .fill(null)
+    .map((_, i) => (i + 1) % max);
+};
+
+export default function InputPanel() {
```

Comment:
suggestion: change variable InputPanel name by a more description name.

Reply:
We will consider this, however, it was the component name suggested by this project. But, I agree that maybe we could review this.